### PR TITLE
refactor: smart_scripts table

### PR DESCRIPTION
**Pull request by BarbzYHOOL**
Just a refactor from HTML to markdown, a bit annoying to do but doable

i used this https://www.tablesgenerator.com/markdown_tables and also i used my text manipulation skills to do it ^^

this is just a test and to show people how we can do it

### CODE REVIEW

File: docs/smart_scripts.md [hunk 1]
```diff
@@ -4,410 +4,50 @@
...
-<td><p>NULL</p></td>
-<td><p><br />
-</p></td>
-<td><p>Event Comment</p></td>
-</tr>
-</tbody>
-</table>
+| Field                 | Type        | Attributes | Key | Null | Default | Extra | Comment       |
+|-----------------------|-------------|------------|-----|------|---------|-------|---------------|
+| [entryorguid][1]      | int(11)     | signed     | PRI | NO   | NULL    |       |               |
```
Comment (Kitzunu) [inline docs/smart_scripts.md:9]: Shouldn't this be [entryorguid](#entryorguid)?

File: docs/smart_scripts.md [hunk 2]
```diff
@@ -4,410 +4,50 @@
...
+[2]: #source_type
+[3]: #id
+[4]: #link
+[5]: #event_type
+[6]: #event_phase_mask
+[7]: #event_chance
+[8]: #event_flags
+[9]: #action_type
+[10]: #target_type
+[11]: #comment
```
Comment (Kitzunu) [inline docs/smart_scripts.md:50]: and this? O.o

Comment (BarbzYHOOL) [reply docs/smart_scripts.md:50]: if you add (#event_phase_mask) then it offsets everything to the right by 20  characters, so here I did it in a clean way, using pro markdown techniques

Comment (Kitzunu) [reply docs/smart_scripts.md:50]: I have no idea how that will turn out, guess time will tell :P

Comment (BarbzYHOOL) [reply docs/smart_scripts.md:50]: it's perfect, however you can not reuse the link called "1" "2" etc, so if there are two tables, then need to use another number (or use letters or whatever)

Comment (BarbzYHOOL) [reply docs/smart_scripts.md:50]: https://github.com/azerothcore/wiki/blob/a8f1a1707abb14a7874a4cf4dd62779abfc31e1d/docs/smart_scripts.md#entryorguid

**Pull request merged**